xc7/tests/ddr/scripts/test_sdram.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
```

Comment:
All the non-generated python files need root level file comments explaining what they are, and their purpose.

Reply:
Done